elf/lto.cc

```diff
@@ -511,7 +511,8 @@ ObjectFile<E> *read_lto_object(Context<E> &ctx, MappedFile<Context<E>> *mf) {
 
   // claim_file_hook() calls add_symbols() which initializes `plugin_symbols`
   int claimed = false;
-  PluginStatus st = claim_file_hook(file, &claimed);
+  // Temporarily unused.
+  [[maybe_unused]] PluginStatus st = claim_file_hook(file, &claimed);
```

Comment:
We should change this to `Fatal` rather than ignoring an error if assert is not enabled. I'll make such change.

Reply:
OK